src/fixtures/configs/python-transformation.json

```diff
@@ -26,7 +26,7 @@
             {
               "name": "Code Z",
               "script": [
-                "# Sum of natural numbers up to num\n\nnum = 16\n\nif num \u003c 0:\n   print(\"Enter a positive number\")\nelse:\n   sum = 0\n   # use while loop to iterate until zero\n   while(num \u003e 0):\n       sum += num\n       num -= 1\n   print(\"The sum is\", sum)"
+                "# Sum of natural numbers up to num\n\nnum = 16\n\nif num \u003c 0:\n    print(\"Enter a positive number\")\nelse:\n    sum = 0\n    # use while loop to iterate until zero\n    while (num \u003c 0):\n        sum += num\n        num -= 1\n    print(\"The sum is\", sum)"
```

Comment:
Testovaci Python kod mal zle odsadenie, napr. 3x space, namiesto 4x space.
Vsimol som si to pri dalsej praci, ked som mal na disku blok transformacie `code.py` a editor mi ho preformatoval.